packages/editor/src/components/editor-canvas/index.js

```diff
@@ -135,6 +136,7 @@ function EditorCanvas( {
 			wrapperBlockName: _wrapperBlockName,
 			wrapperUniqueId: getCurrentPostId(),
 			deviceType: getDeviceType(),
+			hasHistory: !! getEditorSettings().goBack,
```

Comment:
There's already an `editorSettings` variable on line 114, so it could be changed to:
```suggestion
			hasHistory: !! editorSettings.goBack,
```

A similar improvement could happen on line 129.

Reply:
Thanks. Fixed.